src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -167,8 +167,15 @@ bool TuneParamsSelector::VerifyTuneParams(const fully_connected_params& params,
         return false;
 
     auto batch_size = params.is_shape_agnostic ? Align(output_b, tparams.tile_b) : output_b;
-    if (batch_size % (tparams.tile_b * tparams.dispatch_bsv) != 0)
-        return false;
+    if (batch_size % (tparams.tile_b * tparams.dispatch_bsv) != 0) {
+        size_t tile = simd;
+        while (batch_size % tile != 0)
+            tile--;
+        if ((tparams.dispatch_bsv != 1) || (tile > 1) || batch_size == 1)
```

Comment:
maybe can move (tparams.dispatch_bsv != 1)  and batch_size == 1 (if these are needed) a few lines ahead to avoid tile check loop?